### PR TITLE
[CAS-658] Documentation for MessageListView

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
[CAS-658](https://stream-io.atlassian.net/browse/CAS-658)

### Description

Adding code snipets for `MessageListView` in the documentation and creating the new documentation for `MessageListView`. 

https://getstream.io/nessy/docs/chat_docs/android_chat_ux/message_list_view_new

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- ~Changelog updated with client-facing changes~
- ~New code is covered by unit tests~
- ~Comparison screenshots added for visual changes~
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/Android.java [hunk 1]
```diff
@@ -144,4 +159,74 @@ public void navigateTo() {
             ChatUI.instance().getNavigator().navigate(destination);
         }
     }
+
```
Comment (ogkuzmin) [inline stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/Android.java:162]: Need a link to the doc section

**Review by ogkuzmin — COMMENTED** [review]
Why do code snippets differ from documentation provided by link?

**Comment by adasiewiczr** [comment]
Can you update the docs on CMS to match the latest changes (_MessageListView_ has a new package)

**Comment by leandroBorgesFerreira** [comment]
new package added in the CMS

**Review by ogkuzmin — APPROVED** [review]

**Review by adasiewiczr — APPROVED** [review]

**Pull request merged**